packages/playground/src/components/manage_gateway_dialog.vue

```diff
@@ -41,16 +41,16 @@
           </template>
         </v-alert>
 
-        <v-dialog v-model="failedDomainDialog" max-width="400px">
+        <v-dialog v-model="failedDomainDialog" max-width="400px" scrollable>
           <v-card>
             <v-card-title class="bg-warning">Failed Domains</v-card-title>
 
             <v-card-text>
-              <v-list lines="one">
-                <v-list-item v-for="gw in failedToListGws" :key="gw">
-                  <span>* {{ gw }}</span>
-                </v-list-item>
-              </v-list>
+              <ul style="list-style: square">
+                <li v-for="gw in failedToListGws" :key="gw">
+                  <span>{{ gw.slice(prefix.length) }}</span>
```

Comment:
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/5ffe5971-ba16-47dc-9d79-3916d30352bc)

its better to leave it as it is, we have to handle the old gateways or any prefix changes